config/locales/view_components.en.yml

```diff
@@ -12,6 +12,12 @@ en:
       filters:
         filters: Filters
         remove_filters: Clear
+      gantt_chart:
+        task_header: Name
+        today: Today
+        day: Day
+        week: Week
+        month: Month
```

Comment:
We don't need this since we pass a label to the view mode button slot. Or maybe use this as a default label